readme.MD

```diff
@@ -1,39 +1,61 @@
 # Bulletin Board
-RESTful API for a bulletin board.
+
+RESTful API for bulletin board web application.
 
 ## Project configurating and launching:
 
 ### Install
-npm install
 
-### Configurate
-Make a copy of "**ormconfig-example.json**" file and rename it as "**ormconfig.json**"
+```
+$ npm install
+```
+
+### Configure
+
+1. Make a copy of `.env-example` file and rename it as `.env`
+2. Replace `USERNAME`, `PASSWORD`, `HOST_ADDRESS`, `DB_PORT` and `DB_NAME` in `DB_URL` to the ones of your local development environment.
```

Comment:
Hmm for example I don't see `HOST_ADDRESS` in `.env-example`. What about to make this more simple the step 2 just says something like `Replace values to match your local development environment`?

Reply:
The `HOST_ADDRESS` was actually was there, it's all the values in 2. are prefilled in the DB_URL! But I think you're right that the instructions for this don't have to be quite this exact. With the example below it shouldn't be too difficult to figure out how it's done!